packages/pages/src/dev/server/ssr/generateTestData.ts

```diff
@@ -60,6 +60,25 @@ export const generateTestDataForSlug = async (
   const args = getCommonArgs(featuresConfigForEntityPages, projectStructure);
   args.push("--slug", slug);
 
+  const nonStaticFeatureNames = new Set<string>(
```

Comment:
I don't think this is necessary. You can just loop over the templateModuleCollection and add slugField if it's defined, otherwise add entity.slug. It doesn't matter if we loop over static templates too.